Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```
@@ -2074,19 +2074,48 @@
 - type: entity
   name: space cat
   id: MobCatSpace
-  parent: MobCat
+  parent: SimpleSpaceMobBase
   description: Feline pet, prepared for the worst.
   components:
   - type: Sprite
+    drawdepth: Mobs
+    sprite: Mobs/Pets/cat.rsi
     layers:
     - map: ["enum.DamageStateVisualLayers.Base"]
       state: spacecat
+  - type: Physics
```

Turning it into a space mob makes the prototype a bit bloated, in my opinion simply 'disabling' the respirator component by stopping its saturation level from going down would result in less unnecessary copy-pasting.
```suggestion
  - type: Respirator
    minSaturation: 5.0
```

---

I have made your changes and tested them, thank you for your help. o7

---

You're welcome, this was the first commit I felt competent enough to launch a review on o7